napi/src/lib.rs

```diff
@@ -1,5 +1,6 @@
 #![allow(missing_debug_implementations)]
 #![allow(missing_copy_implementations)]
+#![allow(unexpected_cfgs)]
```

Comment:
wasm-bindgen started causing these warnings recently, but it had been fixed in a later version. is this coming from napi_derive? did you look into bumping the dependency?

Reply:
Yeah it's a napi.rs issue, and is fixed, but only in the alpha 3.0 releases.